freeipa.spec.in

```diff
@@ -11,10 +11,7 @@
 # lint is not executed during rpmbuild
 # %%global with_lint 1
 %if 0%{?with_lint}
-    %global enable_pylint_option --enable-pylint
-%else
```

Comment:
As @HonzaCholasta mentioned, if `with_lint` is not defined, linting should be disabled.